pyanaconda/modules/network/installation.py

```diff
@@ -106,6 +120,28 @@ def _write_sysconfig_network(self, root, overwrite):
                                            self.SYSCONF_NETWORK_FILE_PATH),
                                        overwrite)
 
+    def _write_interface_rename_config(self, root, ifname_option_values, overwrite):
+        """Write systemd configuration .link file for interface renaming.
+        :param root: path to the root of the target system
+        :type root: str
+        :param ifname_option_values: list of ifname boot option values
+        :type ifname_option_values: list(str)
+        :param overwrite: overwrite existing configuration file
+        :type overwrite: bool
+        """
+        for ifname_value in ifname_option_values:
+            iface, mac = ifname_value.split(":", 1)
+            content = self.INTERFACE_RENAME_FILE_CONTENT_TEMPLATE.format(mac, iface)
+            file_path = self.INTERFACE_RENAME_FILE_TEMPLATE.format(iface)
```

Comment:
Just an idea.
Wouldn't it be better to have methods to create a string from these templates. These methods could have a documented arguments so it would be easier to not miss `{}` in a templates.